packages/backend/src/api/v1/cells.controller.spec.ts

```
@@ -0,0 +1,63 @@
+        .mockResolvedValue({ id: DOC, workspaceId: WS, type: 'sheet' }),
+    };
+    controller = new ApiV1CellsController(
+      { withDocument } as never,
+      documentService as never,
+    );
+  });
+
+  const lastInitialRoot = () =>
+    withDocument.mock.calls.at(-1)?.[2]?.initialRoot as
+      | SpreadsheetDocument
+      | undefined;
```

ESLint

[error] Replace `|·SpreadsheetDocument⏎·····` with `SpreadsheetDocument`

(prettier/prettier)